evals/refined-templates/validation-template.md

```
@@ -0,0 +1,311 @@
+Set overall_status to:
+- PASS: overall_score >= pass_threshold AND no "BLOCKED" findings
+- NEEDS_REVISION: overall_score < pass_threshold OR non-fatal gaps
+- BLOCKED: severe contradictions that would cause unsafe/wrong implementation (e.g., mutually exclusive behaviors, API described two ways, uninstall semantics contradict CR lifecycle)—even if scores are high
+
+## Scoring math (transparent)
+- completeness_score: 0–100 (if any core completeness pillar is missing, cap at 60 unless user metadata overrides)
+- quality_score: 0–100 from ambiguity/testability/sizing/consistency
+- overall_score: round(0.6 * completeness_score + 0.4 * quality_score) unless user provides weights in metadata; if weights provided, use them and echo in json.metadata.weights_applied
+- overall_status: per rules above vs pass_threshold (default 80)
+
+## Required JSON schema (exact keys)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)